include/mbedtls/psa_util.h

```diff
@@ -293,6 +294,8 @@ static inline int mbedtls_psa_err_translate_pk( psa_status_t status )
             return( MBEDTLS_ERR_ECP_RANDOM_FAILED );
         case PSA_ERROR_BAD_STATE:
             return( MBEDTLS_ERR_PK_BAD_INPUT_DATA );
+        case PSA_ERROR_INVALID_PADDING:
```

Comment:
Do we need to add this here? It seems we have the spacial issue to clean up translation functions (https://github.com/ARMmbed/mbedtls/issues/5514).

Reply:
Yes, I have an implementation of such function to use here in #5519, this should probaly wait until #5519 is merged and re-use it

Reply:
I'll move this as a special return case in `rsa_decrypt_wrap()` and clean this up in #5529

Reply:
I have checked the https://github.com/ARMmbed/mbedtls/pull/5529 and the PR and I think that this one has better chances to go first. Translation function will probably require additional discussion and translation required here is pretty straight forward  `PSA_ERROR_INVALID_PADDING -> MBEDTLS_ERR_RSA_INVALID_PADDING`. When this is merged probably https://github.com/ARMmbed/mbedtls/pull/5529 will have to be rebased.

Sorry for changing my mind, but it seems that this way will be easier.